util/types/json/compare.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2017 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package json
+
+import (
+	"bytes"
+	"reflect"
+	"strings"
+)
+
+// floatEpsilon is for compare two float value allowing precision loss.
```

Comment:
I'm not sure will the folloing description be more explicit.

floatEpsilon is the acceptable error precision when comparing two floating numbers.